Recipes/Recipes/AuthProfile/AuthModel.swift

```diff
@@ -1,4 +1,20 @@
 // AuthModel.swift
 // Copyright © RoadMap. All rights reserved.
 
-struct AuthModel {}
+import Foundation
+
+/// Модель к экрану авторизации
+struct AuthModel {
```

Comment:
```suggestion
struct Auth {
```
Model не пишем это стоп слово